core/src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -1469,6 +1457,7 @@ private boolean postProcessInstance(TreeElement node) {
      * @param dis - the stream to read from.
      */
     public void readExternal(DataInputStream dis, PrototypeFactory pf) throws IOException, DeserializationException {
+        super.readExternal(dis, pf);
         setID(ExtUtil.readInt(dis));
         setName(ExtUtil.nullIfEmpty(ExtUtil.readString(dis)));
         setTitle((String)ExtUtil.read(dis, new ExtWrapNullable(String.class), pf));
```

Comment:
Serialization of `FormDef` is also not covered by the test suites. Might be worth looking into adding it, if it isn't too difficult